### PR TITLE
MM-55443: add server DAU and MAU

### PR DESCRIPTION
**Pull request by ifoukarakis**
#### Summary

- [x] Add DAU & MAU from server reported telemetry.
- [x] Add DAU & MAU from client reported telemetry.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-55443

### CODE REVIEW

File: transform/mattermost-analytics/models/marts/product/features/_product__features__models.yml [hunk 1]
```diff
@@ -42,4 +42,30 @@ models:
...
-        description: The number of unique users related to Guest Accounts feature for the current day and past 29 days.
\ No newline at end of file
+        description: The number of unique users related to Guest Accounts feature for the current day and past 29 days.
+
+      - name: count_unknown_events_daily
+        description: The number of events not in the list of known features for the current day.
+      - name: count_unknown_events_monthly
+        description: The number of events not in the list of known features for the current day and past 29 days.
+      - name: count_unknown_users_daily
+        description: The number of unique users not in the list of known features for the current day.
```
Comment (catalintomai) [inline transform/mattermost-analytics/models/marts/product/features/_product__features__models.yml:52]: maybe rephrase this?

**Review by catalintomai — APPROVED** [review]

**Review by sri-byte — APPROVED** [review]
